[bitnami/clickhouse] add change notes to use Kubernetes 1.8 (#14003)
bitnami/charts: bitnami/clickhouse/README.md

- PV provisioner support in the underlying infrastructure
- ReadWriteMany volumes for deployment scaling

> > **Tip**: If use Kubernetes 1.18, need commented follow code in values

Reviewer: Thanks for your contribution. In order to follow a similar approach to other READMEs in the repository, I proposed some minor changes.
```suggestion
> If you are using Kubernetes 1.18, the following code needs to be commented out.
```
